llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1684,6 +1706,7 @@ void VPlanTransforms::optimize(VPlan &Plan) {
   runPass(removeRedundantInductionCasts, Plan);
 
   runPass(simplifyRecipes, Plan, *Plan.getCanonicalIV()->getScalarType());
+  runPass(simplifyBlends, Plan);
```

Comment:
Maybe we can split out this refactoring and land it first?

Reply:
Good idea, split out in https://github.com/llvm/llvm-project/pull/134073